src/zenml/cli/secret.py

```diff
@@ -121,7 +121,8 @@ def register_secret(
         will interpret the value of the field `password` as the literal string
         `@password`.
 
-        Examples:
+        Examples:-
```

Comment:
```suggestion
        Examples:
```

Reply:
or was that intentional for some reason? I don't get this formatting change

Reply:
The google style guide for docstrings (which we follow), states that a line followed by a colon like this is a section header, but then that means we fail some other rules relating to how a section should be formatted. I've made a fix for this now.